src/codemirror/extensions/highlightLineExtension.ts

```diff
@@ -78,8 +81,14 @@ export interface HighlightLineConfig {
 
 function createCodeMirrorTheme(options: {themeCtx: ThemeContextValue}) {
   const {themeCtx} = options
-  const dark = {color: themeCtx.theme.color.dark[themeCtx.tone]}
-  const light = {color: themeCtx.theme.color.light[themeCtx.tone]}
+
+  // `@sanity/ui@v2.9` introduced two new tones; "neutral" and "suggest",
+  // which maps to "default" and "primary" respectively in the old theme.
+  // The below ensures support with both new and old versions of `@sanity/ui`
+  const fallbackTone = getBackwardsCompatibleTone(themeCtx)
```

Comment:
Id say remove this comment since the function has docs (to avoid comment divergence in the future)